src/settings.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>SUSI Addon Settings</title>
+	<script src="https://use.fontawesome.com/44f32e93e1.js"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="settings.css">
+</head>
+<body id="bdy">
+    <div class="container signin-container">
+<div class="row">
+<div class="col"></div>
+	<div class="col-8">
+ 
+ <div class="card">
+     <img src="images/susi-logo.png" class="img-fluid signin-image">
+	<div class="nologgedin" id="nologgedin">
+		<form id="login" class="col-sm-8" >
+			<input type="email" id="username" placeholder="Email">
+			<br>
+			<input type="password" id="password" placeholder="Password">
+			<br>
+			<br>
+			<button type="submit" name="Login" id="loginbutton" value="Login">Login</button>
+			<br>
+			<p class="col-12" style="margin-left: 100px;">
+               <a href="https://chat.susi.ai/" style="color: #fff; ">New User? Visit our website to sign up for free.</a>
+            </p>
+		</form>
+	</div>
+	<div class="loggedin" id="loggedin">
```

Comment:
The block of code down from here is unnecessary as it doesnt provide any functionality! 
Please remove it!

Reply:
@ms10398 the code below line 31 provides the logout button and links the js files.So what should i remove?

Reply:
Yeah but as that button doesnt provides any functionality thats why i said!